core/src/main/scala/org/broadinstitute/dsde/rawls/submissions/SubmissionsService.scala

```diff
@@ -833,22 +868,25 @@ class SubmissionsService(
         submissionEntity = submissionEntityOpt,
         workflows = workflows ++ workflowFailures,
         status = SubmissionStatuses.Submitted,
-        useCallCache = submissionRequest.useCallCache,
-        deleteIntermediateOutputFiles = submissionRequest.deleteIntermediateOutputFiles,
         submissionRoot = submissionRoot,
-        useReferenceDisks = submissionRequest.useReferenceDisks,
-        memoryRetryMultiplier = submissionRequest.memoryRetryMultiplier,
         workflowFailureMode = workflowFailureMode,
         externalEntityInfo = for {
           entityType <- header.entityType
           dataStoreId <- header.entityStoreId
         } yield ExternalEntityInfo(dataStoreId, entityType),
         userComment = submissionRequest.userComment,
-        ignoreEmptyOutputs = submissionRequest.ignoreEmptyOutputs,
         monitoringScript = submissionRequest.monitoringScript,
         monitoringImage = submissionRequest.monitoringImage,
         monitoringImageScript = submissionRequest.monitoringImageScript,
-        perWorkflowCostCap = submissionRequest.perWorkflowCostCap
+        options = SubmissionOptions(
+          useCallCache = submissionRequest.useCallCache,
+          deleteIntermediateOutputFiles = submissionRequest.deleteIntermediateOutputFiles,
+          useReferenceDisks = submissionRequest.useReferenceDisks,
+          memoryRetryMultiplier = submissionRequest.memoryRetryMultiplier,
+          ignoreEmptyOutputs = submissionRequest.ignoreEmptyOutputs,
+          perWorkflowCostCap = submissionRequest.perWorkflowCostCap
+        ),
+        submissionEntities = submissionEntitiesOpt
```

Comment:
Since we are updating the API, why not replace `submissionEntity` with `submissionEntities` to simplify the implementation and avoid handling both? We could even make it a `v2` api

Reply:
If we just replaced `submissionEntity` with `submissionEntities` it would need to be a V2 in order to not break usage (adding in a new field is fine but removing one will break things).  I did think about doing a V2 but went with this method instead - you'd have to switch over both making a submission and fetching them to V2 in concert to not get weirdness or errors, and that seems harder to coordinate or mess up, but I could be convinced that a V2 is the way to go.